logic-and-algorithm-test/3.py

```diff
@@ -1,25 +1,25 @@
 def generate_sequences(n_terms):
     # Sequence 1: Squares of natural numbers
-    seq1 = [i**2 for i in range(1, n_terms + 1)]
+    sequence1 = [i**2 for i in range(1, n_terms + 1)]
```

Comment:
Renaming `seq1` to `sequence1` enhances clarity by being more descriptive. This is a good practice for improving code readability.